app/routes.py

```diff
@@ -1 +1,102 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, jsonify, abort, make_response, request
+from app.models.task import Task
+from app import db
+from sqlalchemy import asc
+from datetime import datetime
+from datetime import timezone
+import requests
+from dotenv import load_dotenv
+import os
+load_dotenv()
+
+task_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+def validate_model(cls, model_id):
```

Comment:
`validate_model` looks identical to the one in the goal routes file, so it's a great candidate for being refactored into its own file and imported as-needed.